recipe/meta.yaml

```
@@ -285,6 +315,70 @@ outputs:
         https://docs.nvidia.com/cuda/cuquantum/license.html
       doc_url: https://docs.nvidia.com/cuda/cuquantum/latest/custatevec/
       dev_url: https://github.com/NVIDIA/cuQuantum
+  
+  - name: cuquantum-python
+    # cuquantum python version label is MAJOR.MINOR.PATCH
+    version: {{ ".".join(sdk_version[:3]) }}
+    build:
+      number: {{ build_num }}
+      skip: true  # [py<310 or py>312]
+      # build cuQuantum Python package is CUDA agnostic, serving both CUDA 11/12
+      # only build in the CUDA 11 pipeline
+      # (note: this still uses cuda_compiler_version to calculate hash)
+      skip: true  # [not (cuda_compiler_version or "").startswith("11")]
```

```suggestion
      skip: true  # [not (cuda_compiler_version or "").startswith("12")]
```
If possible, I suggest building the pythons on the CUDA 12 runners because the CUDA 11 runners have longer runner setup times for the cudatoolkit since they use RPMs instead of conda packages.